Pocket/pocket-ios
# Put back classic icon with the end of Pride Month

## Goal
* Change the default icon back to the classic logo
* This PR also updates dependencies to the latest

## Test Steps
* Make sure the default logo is the classic one
* Make sure the icon switcher works as expected
